web/src/context/Population.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+  import {
+    FillLayer,
+    hoverStateFilter,
+    LineLayer,
+    VectorTileSource,
+  } from "svelte-maplibre";
+  import { SequentialLegend } from "svelte-utils";
+  import { makeRamp, Popup } from "svelte-utils/map";
+  import { HelpButton, layerId } from "../common";
+  import { assetUrl } from "../stores";
+
+  let showSIMD = false;
+  let showDensity = false;
+
+  // Color ramp from https://www.ons.gov.uk/census/maps/choropleth, dark to light.
+  let simdColorScale = ["#080C54", "#186290", "#1F9EB7", "#80C6A3", "#CDE594"];
+  // The percentiles are [1, 100]. The 5 colors cover 4 each.
+  let simdLimits = [0, 20, 40, 60, 80, 100];
+
+  let densityColorScale = simdColorScale.toReversed();
+  // Use the same (slightly rounded) buckets as https://www.ons.gov.uk/census/maps/choropleth/population/population-density/population-density/persons-per-square-kilometre. TODO Adapt for Scotland.
+  let densityLimits = [0, 4700, 13000, 33000, 94000, 1980000];
```

Comment:
I lifted these from England, but since Scotland has lower density, we could maybe adjust